app/domain/authentication/authn_jwt/consts.rb

```diff
@@ -31,7 +31,7 @@ module AuthnJwt
     CACHE_MAX_CONCURRENT_REQUESTS = 3
     MANDATORY_CLAIMS = [EXP_CLAIM_NAME].freeze
     OPTIONAL_CLAIMS = [ISS_CLAIM_NAME, NBF_CLAIM_NAME, IAT_CLAIM_NAME].freeze
-    VALID_CLAIM_NAME_REGEX = /^[a-zA-Z|$|_][a-zA-Z|$|_|0-9|.]*$/.freeze # starts with letter $ or _ can contains digits and dot
+    VALID_CLAIM_NAME_REGEX = /^[a-zA-Z|$|_][a-zA-Z|$|_|0-9|.\/]*$/.freeze # starts with letter $ or _ can contains digits and dot
```

Comment:
change comment as well

Reply:
@sashaCher WDYT to leave the previous `VALID_CLAIM_NAME_REGEX` logic and to add a command class which handle a nested claim name?
And this class responsibility will be to split the claim name by the delimiter  / and maybe to throw an error about invalid format

Reply:
@nessiLahav See now
I think that creating separated class for nested validation is over-architecture and will make code more complicated.
The responsibility of the class it "claim related variables" input validation rather "single claim section" input validation.
You're welcome to propose a new name for the class reflects the class responsibility.